recipes/qxmpp/all/conanfile.py

```diff
@@ -0,0 +1,71 @@
+from conans import ConanFile, CMake, tools
+from os import rename
+from os.path import join
+
+
+class QxmppConan(ConanFile):
+    name = "qxmpp"
+    license = "LGPL-2.1"
+    homepage = "https://github.com/qxmpp-project/qxmpp"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Cross-platform C++ XMPP client and server library. It is written in C++ and uses Qt framework."
+    topics = ("qt", "qt6", "xmpp", "xmpp-library", "xmpp-server", "xmpp-client")
+    settings = "os", "compiler", "build_type", "arch"
+    exports_sources = ["patches/*"]
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "with_gstreamer": [True, False]}
+    default_options = {"shared": False,
+                       "fPIC": True,
+                       "with_gstreamer": False}
+    generators = "cmake", "cmake_find_package_multi"
+    _cmake = None
+
+    def requirements(self):
+        self.requires("qt/6.1.2")
+        if self.options.with_gstreamer:
+            self.requires("gstreamer/1.19.1")
+            self.requires("glib/2.68.3")
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+            
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self.name)
+
+    def __get_option_str(self, b: bool) -> str:
+        if b:
+            return "ON"
+        else:
+            return "OFF"
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["BUILD_DOCUMENTATION"] = "OFF"
+        self._cmake.definitions["BUILD_TESTS"] = "OFF"
+        self._cmake.definitions["BUILD_EXAMPLES"] = "OFF"
+        self._cmake.definitions["WITH_GSTREAMER"] = self.__get_option_str(self.options.with_gstreamer)
+        self._cmake.configure(source_folder="qxmpp")
```

Comment:
```suggestion
        self._cmake.configure()
```

Use the cmake file wrapper.